DotnetRuntimeMetadata.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Security.Principal.dll; SideIndicator==>}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 119
119:         $diffs | Should -BeExactly $null
```